sdk/search/perf-tests/search-documents/test/getDocumentsCount.spec.ts

```diff
@@ -0,0 +1,251 @@
+              searchable: true,
+              filterable: true,
+              sortable: true,
+              facetable: true
+            },
+            {
+              type: "Edm.String",
+              name: "postalCode",
+              searchable: true,
+              filterable: true,
+              sortable: true,
+              facetable: true
+            }
+          ]
+        },
+        {
+          type: "Collection(Edm.ComplexType)",
+          name: "rooms",
+          fields: [
+            {
+              type: "Edm.String",
+              name: "description",
+              searchable: true,
+              analyzerName: KnownAnalyzerNames.EnLucene
+            },
+            {
+              type: "Edm.String",
+              name: "descriptionFr",
+              searchable: true,
+              analyzerName: KnownAnalyzerNames.FrLucene
+            },
+            {
+              type: "Edm.String",
+              name: "type",
+              searchable: true,
+              filterable: true,
+              facetable: true
+            },
+            {
+              type: "Edm.Double",
+              name: "baseRate",
+              filterable: true,
+              facetable: true
+            },
+            {
+              type: "Edm.String",
+              name: "bedOptions",
+              searchable: true,
+              filterable: true,
+              facetable: true
+            },
+            {
+              type: "Edm.Int32",
+              name: "sleepsCount",
+              filterable: true,
+              facetable: true
+            },
+            {
+              type: "Edm.Boolean",
+              name: "smokingAllowed",
+              filterable: true,
+              facetable: true
+            },
+            {
+              type: "Collection(Edm.String)",
+              name: "tags",
+              searchable: true,
+              filterable: true,
+              facetable: true
+            }
+          ]
+        }
+      ],
+      suggesters: [
+        {
+          name: "sg",
+          searchMode: "analyzingInfixMatching",
+          sourceFields: ["description", "hotelName"]
+        }
+      ],
+      scoringProfiles: [
+        {
+          name: "nearest",
+          functionAggregation: "sum",
+          functions: [
+            {
+              type: "distance",
+              fieldName: "location",
+              boost: 2,
+              parameters: {
+                referencePointParameter: "myloc",
+                boostingDistance: 100
+              }
+            }
+          ]
+        }
+      ],
+      corsOptions: {
+        // for browser tests
+        allowedOrigins: ["*"]
+      }
+    };
+    await this.searchIndexClient.createIndex(hotelIndex);
+  }
+
+  public async globalCleanup() {
+    await this.searchIndexClient.deleteIndex(this.indexName);
+  }
+
+  async runAsync(): Promise<void> {
+    await this.searchClient.getDocumentsCount();
```

Comment:
does the response affect based on the number of documents there are?

Reply:
Yes. But, the documents count will not change in this case. Because, I am creating the index and not actually adding any documents.

Reply:
Sorry, I meant to ask..

does the response time affect based on the number of documents there are?

Reply:
This test has been removed. The requirements is slightly modified. Now, the requirement is to replicate the tests from the Java repository from https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/search/azure-search-perf/src/main/java/com/azure/search/perf

Reply:
But the answer to your question is No. The result for this API does not depend on the documents size. But, it might impact for other APIs such as suggest (that I have added with the recent commit).